### PR TITLE
feat: webapp commit message support

### PR DESCRIPTION
**Pull request by pavlovcik**
GitHub supports suggestions during the pull request review process. This process to make the change to the code, and push the commit is entirely inside of the GitHub pull request review user interface. 

The default git message begins with "Update " and so we decided to add support for this on our fork. 

More context is available here: https://github.com/ubiquity/ubiquity-dollar/pull/623#pullrequestreview-1378020717

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,14 +10,15 @@
     "@actions/exec": "^1.0.3",
     "@actions/github": "^2.1.1",
     "@types/jest": "^26.0.0",
+    "@vercel/ncc": "^0.36.1",
     "got": "^11.3.0",
-    "lodash.get": "^4.4.2"
+    "lodash.get": "^4.4.2",
+    "typescript": "^5.0.3"
```
Comment (adrians5j) [inline package.json:16]: These deps should go into devDeps, no?

Comment (pavlovcik) [reply package.json:16]: @Steveantor what do you think

Comment (Steveantor) [reply package.json:16]: typescript is required for the build process

Comment (adrians5j) [reply package.json:16]: If it's build-related, this should then be a dev dep.

Comment (Steveantor) [reply package.json:16]: I agree, needs to be moved to dev-deps in that sense.

**Review by adrians5j — APPROVED** [review]

**Comment by Steveantor** [comment]
@adrians5j how do we resolve the `index.js` conflict?

**Comment by adrians5j** [comment]
Please just resolve the merge conflict. It's just dist/build.

Whatever you do, no worries about it, I'll rebuild again before I release.

**Comment by Steveantor** [comment]
> Please just resolve the merge conflict. It's just dist/build.
> 
> Whatever you do, no worries about it, I'll rebuild again before I release.

I'm not sure how to resolve merge conflicts of the build/transpiled files,
maybe u could use `git merge <target_branch> -X theirs`

**Comment by adrians5j** [comment]
Just accept yours. It's not a problem, because I'll do a rebuild once this is merged.

**Comment by Steveantor** [comment]
> I think that's permission to force push @Steveantor



> I think that's permission to force push @Steveantor

I don't have the permission to resolve conflicts, The upstream repo owner needs to accept the incoming changes during the merge.
@pavlovcik

**Comment by pavlovcik** [comment]
Steveantor told me that @adrians5j needs to force merge and that from here its out of his hands.

**Comment by adrians5j** [comment]
I can't do changes in fork branches. Changes can be done only on your side.

**Comment by pavlovcik** [comment]
> I can't do changes in fork branches. Changes can be done only on your side.

You have two options:

1. Press "." on your keyboard to edit this pull request in the browser.
2. Or run `gh pr checkout 10` locally.

Screenshot 2023-04-26 at 19 35 00

Sorry mate the road ends here with our team.

**Comment by Steveantor** [comment]
@adrians5j no more merge conflict

**Comment by Steveantor** [comment]
@adrians5j will you look into this PR again?

**Pull request closed without merging**